src/Adapter/Chain/composer.json

```diff
@@ -22,27 +21,25 @@
             "homepage": "https://github.com/nyholm"
         }
     ],
+    "homepage": "http://www.php-cache.com/en/latest/",
     "require": {
         "php": "^5.6 || ^7.0 || ^8.0",
         "cache/adapter-common": "^1.0",
         "cache/tag-interop": "^1.0",
         "psr/cache": "^1.0",
         "psr/log": "^1.0"
```

Comment:
Don't you need to update these too?

Reply:
I hope it will error ? :D

Reply:
No, not really. 
Since we are just consuming the interface (which barely changed between version 1, 2 and 3), I think it will be okey. 
It would have been different if we were implementing the interface.